addOns/pscan/src/main/java/org/zaproxy/addon/pscan/ExtensionPassiveScan2.java

```diff
@@ -194,6 +198,18 @@ public void optionsLoaded() {
         getPassiveScanController();
     }
 
+    /**
+     * Gets the tags used for auto tagging.
+     *
+     * @return a sorted set with the tags, never {@code null}.
```

Comment:
If the user has removed them all will it just be an empty set?

Reply:
Yes.